pkg/solana/logpoller/poller.go

```diff
@@ -0,0 +1,121 @@
+package logpoller
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/base64"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+)
+
+type EventSaver interface {
+	SaveEvent(evt ProgramEvent) error
```

Comment:
This will need to accept a slice of events, more like this:
```suggestion
	SaveEvents(events []ProgramEvent) error
```
But we already have an ORM method that does basically that, so we can probably just use that unless we want to wrap it with some conversion layer:
```
     InsertLogs(logs []Log) error
```

Reply:
That works. This method is only a placeholder and can use the ORM method instead.